app/models/user.rb

```diff
@@ -196,7 +196,7 @@ def mobile_verification_missing?
   end
 
   def image_url
-    identity&.image_url&.gsub('http://', '//')
+    identity&.image_url&.gsub("http://", "//")
```

Comment:
Guard -> rubocop complained about this, no idea what changed because it isn't new code

Reply:
Actually no it was a bug in `.travis.yml` which I've fixed.